docs/alerting/monitors.md

```diff
@@ -41,6 +41,41 @@ Destination | A reusable location for an action, such as Amazon Chime, Slack, or
 
    This information is stored in plain text in the Elasticsearch cluster. We will improve this design in the future, but for now, the encoded credentials (which are neither encrypted nor hashed) might be visible to other Elasticsearch users.
 
+###  Email as a destination
+To receive an alert notification as an email, you must set up **Email** as the destination **Type** for the alert, and configure **Sender** email addresses and **Recipients** or email groups. You can do this using **Manage senders** and **Manage email groups**. 
+
+#### Manage senders
+Manage senders allows you to configure and manage **Sender** email addresses. Senders are email accounts from where the alert notification is sent to different recipients or email groups. Unless you configure at least one sender account in the settings, the **Sender** dropdown list will be empty. 
+
+To configure a sender email, do the following.
+
+1. Choose **Manage senders**.
+1. In the **Manage email senders** modal window, choose **Add sender**. 
+1. Next, choose **New sender**. Multiple senders can be added one at a time.
+1. Enter a unique **Sender name**. Duplicates are not allowed.
+1. Enter the **Email address**, SMTP **Host** (e.g. `smtp.gmail.com` for a Gmail account), and the **Port** number.
```

Comment:
Can people actually use Gmail in this case? Is there a common port we can recommend?

Reply:
Each email account has its own recommended port. E.g. there are separate ports for a Gmail, outlook, hotmail etc. The user needs to know the recommended port for their specific account.